osu.Game/Overlays/ChatOverlay.cs

```diff
@@ -356,10 +357,24 @@ private void postMessage(TextBox textbox, bool newText)
 
             if (postText[0] == '/')
             {
-                // TODO: handle commands
-                currentChannel.AddNewMessages(new ErrorMessage("Chat commands are not supported yet!"));
-                textbox.Text = string.Empty;
-                return;
+                // TODO: Add more commands
+                if (postText.StartsWith("/me "))
+                {
+                    postText = postText.Remove(0,4);
+                    postIsAction = true;
+                } else if (postText == "/me") {

```

Comment:
please format your code in line with the rest of the project. we don't ever put `else if` on the same line as braces